src/statmech.jl

```diff
@@ -13,31 +13,25 @@ arguments:
 - `ionization_energies` is a Dict mapping elements to their first three ionization energies
 - `partition_funcs` is a Dict mapping species to their partition functions
 """
-function saha_ion_weights(T, nₑ, atom::String, ionization_energies::Dict, partition_funcs::Dict)
+function saha_ion_weights(T, nₑ, atom, ionization_energies, partition_funcs::Dict)
```

Comment:
Github isn't letting me directly suggest this (since it hasn't been modified), but could you modify update the definition of the `atom` argument. Maybe you can make it say:

```
- atom, the atomic number or Formula of the element
```

(I would definitely err on the side of being too explicit here, because I think that the code can be a little hard to follow)

Reply:
It can't actually be a `Formula`, right?